Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -1,1452 +1,5555 @@
+        excluded_asset_groups_list = [int(asset_id) for asset_id in argToList(excluded_asset_groups)]
+
+    if excluded_targets is not None:
+        excluded_targets_list = argToList(excluded_targets)
+
+    if frequency is not None:
+        frequency_enum = RepeatFrequencyType[frequency]
+
+    if included_asset_groups is not None:
+        included_asset_groups_list = [int(asset_id) for asset_id in argToList(included_asset_groups)]
+
+    if included_targets is not None:
+        included_targets_list = argToList(included_targets)
+
+    duration_days_int = arg_to_number(duration_days, required=False)
+    duration_hours_int = arg_to_number(duration_hours, required=False)
+    duration_minutes_int = arg_to_number(duration_minutes, required=False)
+    interval_int = arg_to_number(interval, required=False)
+    date_of_month_int = arg_to_number(date_of_month, required=False)
+
+    if enabled is not None:
+        enabled_bool = argToBoolean(enabled)
+
+    else:
+        enabled_bool = False
+
+    duration = generate_duration_time(
+        days=duration_days_int,
+        hours=duration_hours_int,
+        minutes=duration_minutes_int,
+    )
+
+    response_data = client.update_site_scan_schedule(
+        site_id=site.id,
+        scan_schedule_id=scan_schedule_id,
+        enabled=enabled_bool,
+        repeat_behaviour=repeat_behaviour,
+        start_date=start_date,
+        excluded_asset_groups=excluded_asset_groups_list,
+        excluded_targets=excluded_targets_list,
+        included_asset_groups=included_asset_groups_list,
+        included_targets=included_targets_list,
+        duration=duration,
+        frequency=frequency_enum,
+        interval=interval_int,
+        date_of_month=date_of_month_int,
+        scan_name=scan_name,
+        scan_template_id=scan_template_id,
+    )
+
+    return CommandResults(
+        readable_output=f"Scan schedule {scan_schedule_id} has been updated.",
+        raw_response=response_data,
+    )
+
+
+def update_vulnerability_exception_expiration_command(client: Client, vulnerability_exception_id: str,
+                                                      expiration_date: str) -> CommandResults:
+    """
+    Update the expiration date of a vulnerability exception.
+
+    Args:
+        client (Client): Client to use for API requests.
+        vulnerability_exception_id (str): ID of the vulnerability exception to update.
+        expiration_date (str): Expiration date to set for the vulnerability exception,
+            formatted in ISO 8601 format.
+    """
+    response = client.update_vulnerability_exception_expiration(
+        vulnerability_exception_id=vulnerability_exception_id,
+        expiration_date=expiration_date,
+    )
+
+    return CommandResults(
+        readable_output=f"Successfully updated expiration date "
+                        f"of vulnerability exception {vulnerability_exception_id}.",
+        raw_response=response,
+  
```

Comment:
Most of the command calls can be simplified to **args. For example, I'm not sure why you need to init `Site` in `main` rather than the `_command` function.
If you have a different arg name than the one used in the python code, then I suggest changing the function arg to match the one in the `yaml` as it'll make the code easier to read and to maintain.